functorch/functorch/_src/aot_autograd.py

```diff
@@ -369,6 +375,7 @@ def backward(ctx, *flat_args):
     return CompiledFunction.apply
 
 
+@dynamo_timed
```

Comment:
@Chillee any other functions in here worth annotating?

keep in mind that i annotated fw/bw compiler in inductor specifically.  it might be better instead to not annotate the backend function but to annotate the calling of them here, but that might require another layer of wrapping or a contextmgr instead of decorator approach..

Reply:
No this seems fine to me.